w3c/aria-at
# UX changes for undesirable behaviors and remove top level "All assertions are met" radio option

[Preview Tests](https://raw.githack.com/w3c/aria-at/fix-undesirable-validation/index.html)

[Example test](https://raw.githack.com/w3c/aria-at/fix-undesirable-validation/tests/checkbox/test-04-navigate-to-checked-checkbox-reading.html?at=jaws)

To test validation:
1. Select "yes there are undesirable behaviors" for any command for any tests
2. Click the "submit results" button at the bottom of the page
3. You should see "required" by the "undesirable results" fieldset legend for the command in step (1)
4. Select "other"
2. Click the "submit results" button at the bottom of the page
3. You should see "required" by the "undesirable results" for the "other" input for the command

To test tab stop changes:
1. Select "yes there are undesirable changes"
2. Tab into the undesirable checkbox list
3. A second tab should move you out of the the undesirable checkbox list
4. Shift-tab back in
5. Use the arrow keys to move between the undesirable checkboxes
6. Tab out and tab back, the last checkbox with focus should be in the tab sequence
